scripts/performance/perf_processes.py

```diff
@@ -751,6 +842,8 @@ def req_send(self, start_new_batch: bool = False):
         self._closing = True
         if len(self._send_q) > 0:
             await asyncio.gather(*self._send_q, return_exceptions=True)
+        if len(self._gen_q) > 0:
```

Comment:
Why do we need this line now and didn't need it before?

Reply:
this line was added to prevent crashing during the finalization step in case of time consuming requests like revoc_reg